vsts/pipelines/_buildTemplate.yml

```diff
@@ -1,4 +1,12 @@
 steps:
+- task: DownloadPipelineArtifact@0
```

Comment:
q: why do we need this task?

Reply:
we need this when we do a sign build, we need to download the signed binaries and put them in right context/directory for doing a docker build for build image